hera_cal/abscal.py

```diff
@@ -3693,6 +3694,158 @@ def post_redcal_abscal_run(data_file, redcal_file, model_files, raw_auto_file=No
     return hc
 
 
+def run_model_based_calibration(data_file, model_file, output_filename, auto_file=None,
```

Comment:
can you also include a pre-calibration input, that multiplies the data_file before calibration starts (and the output calibration should be the delta between the input calibration and the output data)

Reply:
Will add this.

Reply:
on second thought--I think it is more consistent with other objects in redcal if the output calibiration had the input and delta multiplied. maybe its better to have it this way?

Reply:
Just to clarify, I'm going to multiply the precal gains by the final gain solutions and write those as the output.

Reply:
I've gone and done this.